inference-engine/src/inference_engine/include/openvino/runtime/common.hpp

```diff
@@ -12,6 +12,8 @@
 #include <map>
 #include <string>
 
+#include "ie_common.h"
```

Comment:
Why do you need to include this header here? Maybe will be better to include it only for places which are required it?

Reply:
removed